### PR TITLE
Spelling suggestions

### PR DESCRIPTION
**Pull request by axtens**
So that first one probably isn't the best and hasn't been applied consistently.

Do you realize/realise how many US vs English spelling issues there are in docs? Which one are you in favor/favour of? Or are you going to keep them all and adopt a regex-ish notation like 'favo[u]r' or 'favou?r' and 'alphabeti[s|z]e'

I watched the re-run of the last meeting: Not only should there be a "How Jeremy says it" button but also a "How Jeremy spells it." (It's also how I spell it, by the way)

I'm fairly sure I missed some other spellos. I seem to recall pressing "i" on an /Analaysers/ or something like that.

I used Linux's aspell and probably ended up with some USisms where it should UKisms and vice versa.

### CODE REVIEW

**Comment by axtens** [comment]
Oh and it's /lasagne/ not /lasagna/

**Comment by angelikatyborska** [comment]
Exercism's language is American English (see: https://github.com/exercism/docs/blob/4a42d33991393642d96baae1fd1d9a0dc280d099/building/markdown/style-guide.md#language)

File: building/tracks/stories/anonymous-functions.bomb-defuser.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 Hello again, Agent Double-Null0111.
 
-The forces of UMBRA are acting up again, and it is up to you to foil their latest scheme. Our intel branch informs us that those Minions are planning to set off a stink-bomb at the Governor's ball.
+The forces of UMBRA are acting up again, and it is up to you to foil their latest scheme. Our Intel branch informs us that those Minions are planning to set off a stink-bomb at the Governor's ball.
```
Comment (iHiD) [inline building/tracks/stories/anonymous-functions.bomb-defuser.md]:
```suggestion
The forces of UMBRA are acting up again, and it is up to you to foil their latest scheme. Our intelligence branch informs us that those Minions are planning to set off a stink-bomb at the Governor's ball.
```

File: building/tracks/config-json.md [hunk 2]
```diff
@@ -238,7 +238,7 @@ Each concept is an entry in the top-level `concepts` array. The following fields
 ## Key features
 
 The language's key features succinctly describe what the most important features of the language are.
-They are intended to upsell the more interesting features of a language to potential students.
+They are intended to up-sell the more interesting features of a language to potential students.
```
Comment (iHiD) [inline building/tracks/config-json.md]:
```suggestion
They are intended to upsell the more interesting features of a language to potential students.
```

File: building/markdown/markdown.md [hunk 3]
```diff
@@ -60,7 +60,7 @@ Which renders as:
 
 - The `printf()` function writes to the console.
 
-More complex code (e.g. multiline code) should be wrapped in triple backticks. A [language identifier](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml) should be specified after the opening triple backticks to enable syntax highlighting:
+More complex code (e.g. multi-line code) should be wrapped in triple backticks. A [language identifier](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml) should be specified after the opening triple backticks to enable syntax highlighting:
```
Comment (iHiD) [inline building/markdown/markdown.md]:
```suggestion
More complex code (e.g. multiline code) should be wrapped in triple backticks. A [language identifier](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml) should be specified after the opening triple backticks to enable syntax highlighting:
```

File: building/markdown/internal-linking.md [hunk 4]
```diff
@@ -1,8 +1,8 @@
 # Internal Linking
 
-Links rendered via Exercism's Markdown parser are checked to see if they are internal, and if so may be decorated with tooltips.
+Links rendered via Exercism's Markdown parser are checked to see if they are internal, and if so may be decorated with tool-tips.
 
-A tooltip for a concept might look like this:
+A tool-tip for a concept might look like this:
```
Comment (iHiD) [inline building/markdown/internal-linking.md]:
```suggestion
A tooltip for a concept might look like this:
```

File: building/markdown/internal-linking.md [hunk 5]
```diff
@@ -1,8 +1,8 @@
 # Internal Linking
 
-Links rendered via Exercism's Markdown parser are checked to see if they are internal, and if so may be decorated with tooltips.
+Links rendered via Exercism's Markdown parser are checked to see if they are internal, and if so may be decorated with tool-tips.
```
Comment (iHiD) [inline building/markdown/internal-linking.md]:
```suggestion
Links rendered via Exercism's Markdown parser are checked to see if they are internal, and if so may be decorated with tooltips.
```

**Comment by iHiD** [comment]
@axtens Thanks! I've removed a couple that I disagreed with (e.g. tool-tip being changed to tooltip) and merged the rest :)

**Pull request merged**